scripts/unixbench/cb_unixbench.sh

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2012 IBM Corp.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+set -x
+
+source $(echo $0 | sed -e "s/\(.*\/\)*.*/\1.\//g")/cb_common.sh
+
+LOAD_PROFILE=$1
+LOAD_LEVEL=$2
+LOAD_DURATION=$3
+LOAD_ID=$4
+SLA_RUNTIME_TARGETS=$5
+
+if [[ -z "$LOAD_PROFILE" || -z "$LOAD_LEVEL" || -z "$LOAD_DURATION" || -z "$LOAD_ID" ]]
+then
+    syslog_netcat "Usage: cb_unixbench.sh <load profile> <load level> <load duration> <load_id>"
+    exit 1
+fi
+
+UNIXBENCH_DIR="~/byte-unixbench/UnixBench"
+eval UNIXBENCH_DIR=${UNIXBENCH_DIR}
+
+UNIXBENCH_IP=`get_ips_from_role unixbench`
+source ~/cb_barrier.sh start
```

Comment:
Remove this barrier line, please. It's a duplicate. I've a PR coming that also fixes the duplicates in other workloads.